queries/liquid/highlights.scm

```diff
@@ -0,0 +1,123 @@
+([
+  "{{"
+  "}}"
+  "{{-"
+  "-}}"
+  "{%"
+  "%}"
+  "{%-"
+  "-%}"
+] @tag.delimiter
+  (#set! priority 110))
+
+([
+  "]"
+  "["
+  ")"
+  "("
+] @punctuation.bracket
+  (#set! priority 110))
+
+([
+  ","
+  "."
+] @punctuation.delimiter
+  (#set! priority 110))
```

Comment:
Im pretty sure these one have the same highlights as js and html, so probably unnecessary to add priorities to these